src/Features/Core/Portable/GenerateMember/GenerateParameterizedMember/AbstractGenerateDeconstructMethodService.State.cs

```diff
@@ -7,10 +7,14 @@
 using System.Collections.Immutable;
 using System.Threading;
 using System.Threading.Tasks;
+using System.Xml.Linq;
```

Comment:
I don't think this namespace is used. Could you clean up the unused `usings`?

Reply:
Yes, I will do that!